index.html

```diff
@@ -34,12 +34,236 @@ <h3>I am a software developer</h3>
             <div class="mess"><img src="./images/Icon - Medium.png" alt=""></div>
         </section> 
        </section>
-
-       <section>
-        <div>
-            <img src="./images/Icon - Angelist.png" alt="">
+       <section class="recent">
+        <div class="title">
+            <h2>My Recent works</h2>
+            <div class="v"></div>
+        </div>
+        <div class="frame">
+            <div class="featured">
+                <div class="imgp"></div>
+                <div class="block">
+                    <p class="post">Multi-Post Stories</p>
+                    <p class="support">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                    <section class="tag-main">
+                        <ul class="tag1">
+                            <li>css</li>
+                        <ul class="tag2">
+                            <li>html</li>
+                        </ul>
+                        <ul class="tag3">
+                            <li>bootstrap</li>
+                        </ul>
+                        <ul class="tag4">
+                            <li>Ruby</li>
+                        </ul>
+                    </section>
+                    <section class="buttom">
+                        <p class="see">See project</p>
+                    </section>
+                </div>
+            </div>
+            <section class="frame-sub">
+                <section class="card-works">
+                    <section class="mask">
+                        <section class="img-placeholder">
+                            <section class="div">
+                                <section class="right-works">
+                                    <p class="pro">Profesional Art Printing Data</p>
+                                    <p class="sub1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                                    <ul class="tags">
+                                        <li class="tags1"><h6>html</h6></li>                                            
+                                        <li class="tags2"><h6>bootstrap</h6></li>
+                                        <li class="tags3"><h6>Ruby</h6></li>
+                                    </ul>
+                                    </section>
+                                </section>
+                            </section>
+                        </section>
+                        <!-- <section class="mask-sub"></section> -->
+                    </section>
+                    <section class="action">
+                        <section class="but">
+                            <p class="see1">See Project</p>
```

Comment:
- [ ] As the last code reviewer said: Please 🙏🏽 use the `button` tag or the `a` tag for all your buttons. This is the best HTML practice.